testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/naming/remote/ejb/RemoteNamingEjbTestCase.java

```diff
@@ -138,10 +139,11 @@ public void testDeploymentBinding() throws Exception {
             try {
                 ctx.lookup("some/entry");
                 fail("expected exception");
-            } catch (NameNotFoundException e) {
+            } catch (NamingException e) {
```

Comment:
I don't think this exception needs to be changed. A NameNotFoundException should still be thrown (see https://github.com/wildfly/wildfly-naming-client/pull/9).